x-pack/plugins/observability/public/hooks/slo/use_fetch_slo_list.ts

```diff
@@ -70,6 +72,7 @@ export function useFetchSloList({
               ...(sortBy && { sortBy }),
               ...(sortDirection && { sortDirection }),
               ...(page && { page }),
+              ...(search && { search }),
```

Comment:
This search query parameter does not exist on this API, we need to use the kqlQuery instead, which can be: `slo.name : "My SLO"`